railib/api.py

```diff
@@ -22,6 +22,8 @@
 
 from . import rest
 
+from requests_toolbelt import MultipartDecoder
```

Comment:
Did you try to see if it's easy to get rid of this dependency?

Reply:
@geokollias I am looking into it.

Reply:
@geokollias I am able to parse the response/parts by decoding it using the email.parser.BytesParser() and splitting the result on the form-data boundary. Currently I am having issues with decoding the arrow stream because somehow BytesParser is adding some null chars in the response, which is failing the arrow library. I think using requests_toolbelt is fine because it makes it easy to read and manipulate the parts. As the response parsing is handling on the client/user's part in our SDK (like we do in examples), it will be easy for the user to parse the toolbelt's parts objects rather than doing string manipulation. What do you think? If you are ok with toolbelt then we can go ahead with this example.